imports/message-read-receipt/server/hooks.js

```diff
@@ -14,6 +14,10 @@ callbacks.add('afterSaveMessage', (message, room) => {
 
 	// mark message as read as well
 	ReadReceipt.markMessageAsReadBySender(message, room._id, message.u._id);
+
+	if (message.t === 'livechat-close') {
```

Comment:
This is not a good idea.
Let's check if the room is not closed before calling the method, like this:
```
	if (room && !room.closedAt) {
		Subscriptions.setAsReadByRoomIdAndUserId(room._id, message.u._id);
	}
```

But, to make it works fine, you will need to send an updated room object here:
 https://github.com/RocketChat/Rocket.Chat/blob/develop/app/livechat/server/lib/Livechat.js#L317

Reply:
ok, let me update.